api/src/main/java/com/epam/pipeline/manager/pipeline/runner/PipeRunCmdBuilder.java

```diff
@@ -130,49 +133,50 @@ public PipeRunCmdBuilder config() {
 
     public PipeRunCmdBuilder yes() {
         if (startVO.isYes()) {
-            cmd.add("-y");
+            addCmd("-y");
         }
         return this;
     }
 
     public PipeRunCmdBuilder quite() {
         if (startVO.isQuite()) {
-            cmd.add("-q");
+            addCmd("-q");
         }
         return this;
     }
 
     public PipeRunCmdBuilder sync() {
         if (startVO.isSync()) {
-            cmd.add("-s");
+            addCmd("-s");
         }
         return this;
     }
 
     public PipeRunCmdBuilder parameters() {
         if (startVO.isShowParams()) {
-            cmd.add("-p");
+            addCmd("-p");
         }
         return this;
     }
 
     public PipeRunCmdBuilder runParameters() {
         if (MapUtils.isNotEmpty(runVO.getParams())) {
+            addCmd(CMD_OPTIONS_DELIMITER);
             final String parametersCommand = runVO.getParams().entrySet()
                     .stream()
                     .map(this::prepareParams)
-                    .collect(Collectors.joining(WHITESPACE));
+                    .collect(Collectors.joining(WHITESPACE + getNewLineIndicator() + WHITESPACE));
             cmd.add(parametersCommand);
         }
         if (Objects.nonNull(runVO.getParentRunId())) {
-            cmd.add(String.format("parent-id %d", runVO.getParentRunId()));
```

Comment:
If parameters are empty, we need to add '--' here as well

Reply:
added